app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -259,6 +259,30 @@
         } else if (term.indexOf('quality:') >= 0 || term.indexOf('percentage:') >= 0) {
           filter = term.replace('quality:', '').replace('percentage:', '');
           addPredicate("quality", filter);
+        } else if (term.indexOf('rof:') >= 0) {
```

Comment:
Wow, that's a lot of new terms! Maybe we should just have a list and loop over them instead of making so many `else if`s.